<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>506</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="input_type_combobox">
       <item>
        <property name="text">
         <string>File (.txt)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plaintext</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="browse_button">
       <property name="text">
        <string>Browse File</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="input_box"/>
     </item>
     <item row="5" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Modulo key (m)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="m_key_box"/>
     </item>
     <item row="8" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="encrypt_button">
         <property name="text">
          <string>Encrypt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decrypt_button">
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Input Type</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPlainTextEdit" name="output_box"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save Output as file</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="b_key_box"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Shift key (b)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="vigenere_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Affine Cypher Encryption and Decryption</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browse_button</sender>
   <signal>clicked()</signal>
   <receiver>input_box</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_type_combobox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>browse_button</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encrypt_button</sender>
   <signal>clicked()</signal>
   <receiver>output_box</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decrypt_button</sender>
   <signal>clicked()</signal>
   <receiver>output_box</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
